site/components/Navigation/NavItemDesktop.tsx

```diff
@@ -23,7 +23,7 @@ export const NavItemDesktop: FC<Props> = ({
         <Link href={url}>
           <a
             className={cx(styles.navMain_DesktopLink, {
-              dropdown: subMenu !== undefined,
+              dropdown: !!subMenu,
             })}
             data-active={active?.toString()}
             {...props}
```

Comment:
ah see this is a bit weird now because the `props` are typed to inherit `HTML Anchor` props but the anchor tag isn't always rendered. Also per a recent discussion in one of ladytrekkers recent PRs, I've come to dislike spreading `...props` because it can lead to bugs as its easy to accidentally pass down wrong props.

I suggest breaking this whole thing into two separate components, and then narrowing down the props / removing the unnecessary ones
* [ ] `LinkItemDesktop` which renders a nextjs `<Link>`
* [ ] `DropdownItemDesktop` which uses tippy and renders `props.children` in the dropdown
```
<DropdownItemDesktop>
  <LinkItemDesktop />
  <LinkItemDesktop />
</DropdownItemDesktop>
```
If it's too hard to get `LinkItemDesktop` to work as tippy child feel free to make a third component called `DropdownLink`
* [ ] Don't destructure props or spread `...props` down either, don't do `Props extends HTMLAttributes` just be explicit about which props it each one takes and pass them down as needed 🙏

Reply:
Feel free to do this in a follow up PR btw, just make a ticket and tag yourself if you decide to do it separate

Reply:
Sweet, good ideas. Feels cleaner having them separate. TY